frontend/src/lib/components/PropertyFilters/Filter/TabbedPropertyFilter.tsx

```diff
@@ -125,10 +155,11 @@ function CohortPaneContents({ onComplete, setThisFilter, value, displayOperatorA
                     }
                     onChange={(_, newFilter) => {
                         onComplete()
-                        setThisFilter('id', newFilter.value, undefined, newFilter.type)
+                        const { value: filterValue, type } = newFilter as { value: string; type: string }
```

Comment:
`PropertySelect` component is already typed you shouldn't need to specify the extra `as { value: string; type: string }`

Reply:
The type here is `OptionsType | OptionData | OptionGroupData` which comes from Antd. I have no idea why it isn't just OptionsType.

Reply:
Cool, no point in spending too much time here anyways